src/dymaptic.GeoBlazor.Core/package-lock.json

```diff
@@ -8,19 +8,18 @@
       "name": "dymaptic.GeoBlazor.Core",
       "license": "ISC",
       "dependencies": {
-        "@arcgis/core": "4.28",
-        "esbuild": "0.19.5",
-        "protobufjs": "7.2.5",
-        "protobufjs-cli": "1.1.2"
+        "@arcgis/core": "4.28.10",
+        "esbuild": "0.19.8",
+        "protobufjs": "7.2.5"
```

Comment:
so we are just removing the cli? esbuild and arcgis are the updates.

Reply:
protobufjs-cli, yes.